python/lsst/ts/ess/controller/device/vcp_ftdi.py

```diff
@@ -80,6 +79,9 @@ async def basic_open(self) -> None:
         IOError if virtual communications port fails to open.
         """
         self.vcp.open()
+        # Do not move this line to before the instruction to open the device
+        # because doing so will result in an error.
```

Comment:
Why is it an error to set the baud rate after opening the device? Please explain in the comment. Normally one would expect to fully configure a communication channel before opening it!

Reply:
I have improved the comment to explain better why the baud rate can only be set after calling 'open()'.